src/components/DriverCard.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+function DriverCard(props) {
+    const {name,rating,img, car} = props
+
+    const divStyle = {
+    height:200,
+    width:200,
+    }
+
+  return (
+    <div>
+
+    <img src={img} alt={name} style={divStyle}/>
+    <p>{name}</p>
+    <p>{rating}</p>
```

Comment:
Here they ask you to display the rating with stars (★) and not with number. You already have a component that does that so you can import and use it here:
```
import React from 'react'
import Rating from './Rating'

function DriverCard(props) {
    const {name,rating,img, car} = props

    const divStyle = {
    height:200,
    width:200,
    }

  return (
    <div className='drivercard'>

    <img className="circle-image"src={img} alt={name} style={divStyle}/>
    <p>{name}</p>
    <Rating>{rating}</Rating>
    <p>{car.model} - {car.licensePlate}</p>

    </div>
  )
}

export default DriverCard
```